etc/conf/atTelemetry_salobj.yaml

```diff
@@ -0,0 +1,271 @@
+url_format: "http://{ip_address}:{port_number}/{filename_HDR}"
+
+# Default write_mode (fits or string)
+write_mode: fits
+
+# Timeout for waiting for end of Telemetry event (in seconds)
+timeout_endTelem: 120
+
+# Default string delimeter (string mode only)
+hdu_delimiter: END
+
+# The file naming event source and definitions
+filepath: 'www/DMHS_filerepo'
+format_HDR: "{}.header"
+format_FITS: "{}.fits"
+imageName_event:
+    device: ATCamera
+    topic: startIntegration
+    value: imageName
+
+# The Events that we use to define where we collect
+start_collection_event:
+    device: ATCamera
+    topic: startIntegration
+
+end_collection_event:
+    device: ATCamera
+    topic: endOfImageTelemetry
+
+# The event for the image readout paramters
+imageParam_event:
+    device: ATCamera
+    topic: imageReadoutParameters
+
+# Telemetry values for images meta-data
+# https://confluence.lsstcorp.org/pages/viewpage.action?spaceKey=SYSENG&title=Auxiliary+Telescope+Header+Information+Topic+Mapping
+telemetry:
+
+ DATE-OBS:
+  device: ATCamera
+  Stype: Event
+  topic: startIntegration
+  value: timeStamp
+  collect_after_event: start_collection_event
+
+ DATE-BEG:
+  device: ATCamera
+  Stype: Event
+  topic: startIntegration
+  value: timeStamp
+  collect_after_event: start_collection_event
+
+ DATE-END:
+  device: ATCamera
+  Stype: Event
+  topic: endReadout
+  value: timeStamp
+  collect_after_event: end_collection_event
+
+ OBSID:
+  device: ATCamera
+  Stype: Event
+  topic: startIntegration
+  value: imageName
+  collect_after_event: start_collection_event
+
+ SEQNUM:
+  device: ATCamera
+  Stype: Event
+  topic: startIntegration
+  value: imageIndex
+  collect_after_event: start_collection_event
+
+ GROUPID:
+  device: ATCamera
+  Stype: Event
+  topic: startIntegration
+  value: imageSequenceName
+  collect_after_event: start_collection_event
+
+# ----------------------------------------
+# -- still not available from SAL --------
+# OBSTYPE:
+#   device: ATCamera
+#   Stype: Event
+#   topic: startIntegration
+#   value: imageType
+#   collect_after_event: start_collection_event
+# ---------------------------------------------
+
+ EXPTIME:
+  device: ATCamera
+  Stype: Event
+  topic: startIntegration
+  value: exposureTime
+  collect_after_event: start_collection_event
+
+ SHUTTIME:
+  device: ATCamera
+  Stype: Event
+  topic: shutterMotionProfile
+  value: measuredExposureTime
+  collect_after_event: end_collection_event
+
+ CCDTEMP:
+  device: ATCamera
+  Stype: Telemetry
+  topic: wreb
+  value: ccdTemp0
+  collect_after_event: start_collection_event
+
+# --- Pointing Information ---------------
+
+ RA:
+  device: ATPtg
+  Stype: Event
+  topic: currentTarget
+  value: ra
+  collect_after_event: start_collection_event
+
+ DEC:
+  device: ATPtg
+  Stype: Event
+  topic: currentTarget
+  value: declination
+  collect_after_event: start_collection_event
+
+# --- Will come from ATMCS rotator angle when available
+# ROTPA:
```

Comment:
`rotPA` is a field of the target `currentTarget` event. Can you enable this entry?

Reply:
It was deemed inappropriate by Patrick and should be replaced by `ROTSTART/END` from encoders in the near future. Left as a reminder.